sdks/java/io/hcatalog/resources/testconf/hive-log4j2.properties

```diff
@@ -0,0 +1,123 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I have refactored these and do not need hive-log4j2.properties anymore. The only required file is hive-site.xml which I will move inside test/resources.